2021-04-22 07:40:28,007 INFO    MainThread:18796 [wandb_setup.py:_flush():69] setting env: {}
2021-04-22 07:40:28,007 INFO    MainThread:18796 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-22 07:40:28,007 INFO    MainThread:18796 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/code/src/wandb/run-20210422_074028-3fhlzv99/logs/debug.log
2021-04-22 07:40:28,007 INFO    MainThread:18796 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/code/src/wandb/run-20210422_074028-3fhlzv99/logs/debug-internal.log
2021-04-22 07:40:28,007 INFO    MainThread:18796 [wandb_init.py:init():369] calling init triggers
2021-04-22 07:40:28,008 INFO    MainThread:18796 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-22 07:40:28,008 INFO    MainThread:18796 [wandb_init.py:init():418] starting backend
2021-04-22 07:40:28,008 INFO    MainThread:18796 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-22 07:40:28,021 INFO    MainThread:18796 [backend.py:ensure_launched():123] starting backend process...
2021-04-22 07:40:28,030 INFO    MainThread:18796 [backend.py:ensure_launched():128] started backend process with pid: 18821
2021-04-22 07:40:28,032 INFO    MainThread:18796 [wandb_init.py:init():423] backend started and connected
2021-04-22 07:40:28,034 INFO    MainThread:18796 [wandb_init.py:init():465] updated telemetry
2021-04-22 07:40:28,034 INFO    MainThread:18796 [wandb_init.py:init():484] communicating current version
2021-04-22 07:40:28,921 INFO    MainThread:18796 [wandb_init.py:init():489] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-22 07:40:28,921 INFO    MainThread:18796 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-04-22 07:40:29,137 INFO    MainThread:18796 [wandb_init.py:init():522] starting run threads in backend
2021-04-22 07:40:29,168 INFO    MainThread:18796 [wandb_run.py:_console_start():1524] atexit reg
2021-04-22 07:40:29,168 INFO    MainThread:18796 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-22 07:40:29,168 INFO    MainThread:18796 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-22 07:40:29,171 INFO    MainThread:18796 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-22 07:40:29,171 INFO    MainThread:18796 [wandb_init.py:init():546] run started, returning control to user process
2021-04-22 07:40:43,396 INFO    MainThread:18796 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 1
2021-04-22 07:40:43,401 INFO    MainThread:18796 [wandb_run.py:_restore():1466] restore
2021-04-22 07:40:45,612 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 574
}

2021-04-22 07:40:46,153 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 574
  total_bytes: 4683
}

2021-04-22 07:40:46,540 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:46,642 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:46,744 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:46,847 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:46,949 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:47,051 INFO    MainThread:18796 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 4683
  total_bytes: 4683
}

2021-04-22 07:40:48,141 INFO    MainThread:18796 [wandb_run.py:_show_files():1838] logging synced files
